scripts/prepack.js

```diff
@@ -0,0 +1,30 @@
+#!/usr/bin/env node
+
+// This script removes the build artifacts of ignored contracts.
+
+const fs = require('fs');
+const path = require('path');
+const proc = require('child_process');
+
+function exec(cmd, ...args) {
+  proc.execFileSync(cmd, args, { stdio: 'inherit' });
+}
+
+exec('npm', 'run', 'compile');
+
+const ignoredPaths = [
+  path.resolve('contracts/mocks'),
```

Comment:
Can't this be read from the `package.json`?

Reply:
Yes, with some more work, but it's true it will be more robust.